app/models/concerns/turbo/broadcastable.rb

```diff
@@ -40,6 +40,20 @@
 #       end
 #   end
 #
+# If you want to render a template instead of a partial, e.g. ('messages/index' or 'messages/show'), you can use the `template:` option.
+# Again, only to any broadcast method that accepts the `**rendering` argument. Example:
+#
+#   class Message < ApplicationRecord
+#     belongs_to :user
+#
+#     after_create_commit :update_message
+#
+#     private
+#       def update_message
+#         broadcast_replace_to(user, :message, target: "message", template: "messages/show", locals: { message: self })
+#       end
+#   end
```

Comment:
This might not be the best example but I'm just trying to follow the existing "messages" context of the examples.